docs/src/tutorial/pure.rst

```diff
@@ -243,6 +243,13 @@ Further Cython functions and declarations
     print(cython.sizeof(cython.longlong))
     print(cython.sizeof(n))
 
+* ``typeof`` returns a string representation of the passed type.  It can take expressions.
```

Comment:
```suggestion
* ``typeof`` returns a string representation of the argument's type for debugging purposes.  It can take expressions.
```

Reply:
Well, if there's "debugging purpose" in here, then it's fine. We still could make `typeof` that will return type and not effect a lot of code